components/range.js

```diff
@@ -0,0 +1,62 @@
+const template = `
+<div class="c-range" :class="classes">
+  <label class="c-range__label" :for="name">
+    <span dir="ltr">{{label}}</span>
+  </label>
+  <small v-if="hint" class="c-range__hint">
+    <span dir="ltr">
+      {{ hint }}
+    </span>
+  </small>
+  <slot v-else name="hint"></slot>
+  <input class="c-range__input" :id="name" type="range" style="background-size: 50%;">
+  <small v-if="message" class="c-range__message" :class="messageClasses">
```

Comment:
Instead of a single computed property, **messageClasses**, let's turn  **classes** into an object that has a structure of key => object, (i.e: **classes.message**)